MockKitUI/View Controllers/MockCGMManagerSettingsNavigationController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  MockCGMManagerSettingsNavigationController.swift
+//  MockKitUI
+//
+//  Created by Nathaniel Hamming on 2021-01-13.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import LoopKit
+import LoopKitUI
+
+final class MockCGMManagerSettingsNavigationController: SettingsNavigationViewController, PreferredGlucoseUnitObserver {
```

Comment:
For my work, I had to create a `CGMManagerSettingsNavigationController` and `PumpManagerSettingsNavigationController` for my work for other reasons (I needed to add a different delegate protocol to conform to). 

I would suggest renaming this to just `CGMManagerSettingsNavigationController`, make it a generic with a constraint that the `rootViewController` type must conform to `PreferredGlucoseUnitObserver`, make it public (like `SettingsNavigationViewController` is), and use it here and throughout the plugins. (Therefore, no need to have a separate `*SettingsNavigationViewController` for each plugin.)